sources/packages/forms/src/form-definitions/studentdependantsappealparttime.json

```diff
@@ -0,0 +1,970 @@
+{
+    "title": "Appeals - Student Dependants Part Time",
+    "name": "studentDependantsAppealPartTime",
+    "path": "studentdependantsappealparttime",
+    "type": "form",
+    "display": "form",
+    "tags": [
+        "common"
+    ],
+    "components": [
+        {
+            "key": "dependantsInformationTitle",
+            "label": "Dependants Information (Part-Time Applications)",
+            "input": false,
+            "tag": "h3",
+            "attrs": [
+                {
+                    "value": "",
+                    "attr": ""
+                }
+            ],
+            "className": "category-header-medium primary-color",
+            "content": "Dependants Information (Part-Time Applications)",
+            "type": "htmlelement",
+            "hideLabel": true,
+            "tags": [],
+            "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+            },
+            "properties": {},
+            "lockKey": true,
+            "isNew": false
+        },
+        {
+            "clearOnHide": false,
+            "key": "dependantsAppealPanel",
+            "input": false,
+            "title": "Dependants Information Part Time Applications Panel",
+            "theme": "default",
+            "tableView": false,
+            "components": [
+                {
+                    "autofocus": false,
+                    "input": true,
+                    "tableView": true,
+                    "inputType": "radio",
+                    "label": "<strong>Do you have any dependants?</strong>",
+                    "key": "hasDependents",
```

Comment:
For appeal forms whatever existed so far, we prefixed the variables with `studentNew`.

e.g. here `studentNewHasDependents` 

same for `dependants` and `supportnocustodyDependants`

@andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993 should we have the `studentNew` prefix for api keys in appeal forms as standard practice ??

Reply:
I noticed that it was added during the PD/PPD appeal implementation but I do not see a reason why we should do it.
IMO we can keep the same value from the original form, when one is available, keeping in mind that the appeal form can potentially receive inputs that are not even in the application form and that would be one more reason that using "new" would not be advised.

Reply:
Thanks @dheepak-aot and @andrewsignori-aot. I will keep the same keys as they are in the original family information form.

Reply:
I concur with the decision.

Reply:
I agree, let not have the studentNew prefixed